### ISSUE TITLE
Whiteboard did not save properly. As a result, content was lost.

### ISSUE THREAD

**Issue by LuciousRuby** [worldanvil/worldanvil-bug-tracker#1140]
**World Anvil Username**:

**World / Campaign / RPG Character / Manuscript Title**:

**Feature**: 

**Related URLS** (To public & visible pages add screenshots otherwise):

**Describe the Issue**
A clear and concise description of what the issue is.

**To Reproduce**

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!

**Comment by LuciousRuby** [comment]
The whiteboard did not save properly. As a result, I lost my content

**Comment by Kordishal** [comment]
Hi! It is of course unfortunate that you lost your content. The team will need more information to be able to check what is going on.

Please add a link to your whiteboard, your World Anvil user name at least and describe in more detail what exactly happened.

Keep in mind that there is an indicator that tells you when there are unsaved changes, if you reload or close the page before that turns to saved your content will be lost.

**Comment by Duranzno** [comment]
Checking on this

**Comment by LuciousRuby** [comment]
> Hi! It is of course unfortunate that you lost your content. The team will need more information to be able to check what is going on.
> 
> Please add a link to your whiteboard, your World Anvil user name at least and describe in more detail what exactly happened.
> 
> Keep in mind that there is an indicator that tells you when there are unsaved changes, if you reload or close the page before that turns to saved your content will be lost.

**Comment by LuciousRuby** [comment]
> Hi! It is of course unfortunate that you lost your content. The team will need more information to be able to check what is going on.
> 
> Please add a link to your whiteboard, your World Anvil user name at least and describe in more detail what exactly happened.
> 
> Keep in mind that there is an indicator that tells you when there are unsaved changes, if you reload or close the page before that turns to saved your content will be lost.

Here is the whiteboard link: [whiteboard:4bab21e0-be02-4f75-b460-3ac878edf273]

It was saying that it was saving as I was adding to the whiteboard. I went to another page and when I came back, the information was gone.

**Comment by Kordishal** [comment]
I assume you did not close this on purpose. And what you provided is the whiteboard BBCode, but I think that should work too as it contains the identifier.

**Comment by LuciousRuby** [comment]
Hi Kordishal, 

Sorry I am new to this hub and don't really understand how it works. Basically, the whiteboard was saying that it was saving as I was working in it. I clicked away from the whiteboard and when it came back, it was gone. That's all I understand of it. But I did notice it was saying "Saving..." regularly. This is why I thought it was a bug in the system and thought to report it. If I don't get me work back, I will just redo it. But I am sure others will be frustrated.

Thanks

Jeannette
